tests/cpu/fileformats/FileFormatCTF_tests.cpp

```diff
@@ -6229,6 +6229,30 @@ OCIO_ADD_TEST(CTFTransform, gamma6_ctf)
     OCIO_CHECK_EQUAL(expected, outputTransform.str());
 }
 
+OCIO_ADD_TEST(CTFTransform, fixed_function_aces_gamutmap_13)
```

Comment:
Would be nice to check the inverse too (see the Rec2100 surround test, although could also be done within the same test, if you prefer).

Reply:
Added a separate test, thanks for the suggestion.